openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2052,6 +2052,83 @@ regulation), the Credential Issuer should properly authenticate the Wallet and e
   </front>
 </reference>
 
+<reference anchor="W3C.Digital_Credentials_API" target="https://wicg.github.io/digital-credentials/">
+        <front>
+          <title>Digital Credentials API</title>
+		  <author fullname="Marcos Caceres">
+            <organization>Apple Inc.</organization>
+          </author>
+          <author fullname="Tim Cappalli">
+            <organization>Okta</organization>
+          </author>
+          <author fullname="Sam Goto">
+            <organization>Google</organization>
+          </author>
+        </front>
+</reference>
+
+# OpenID4VCI over the Digital Credentials API {#dc_api}
+
+This section defines how to use OpenID4VCI with the Digital Credentials API.
+
+The name "Digital Credentials API" (DC API) encompasses the W3C Digital Credentials API [@!W3C.Digital_Credentials_API]
+as well as its native App Platform equivalents in operating systems (such as [Credential Manager on Android](https://developer.android.com/jetpack/androidx/releases/credentials)).
+The DC API allows web sites and native apps acting as Issuers to initiate the issuance of verifiable credentials.
+The API itself is agnostic to the Credential issuance protocol and can be used with different protocols.
+The Web Platform, working in conjunction with other layers, such as the app platform/operating system, and based on the permission of the End-User, will send the credential offer along with the Origin of the Issuer to the End-User's chosen Wallet.
+
+## Protocol
+
+To use OpenID4VCI with the Digital Credentials API (DC API), the exchange protocol value has the following format: `openid4vci-v<version>`.
+
+The value `1` MUST be used for the `<version>` field to indicate the request and response are compatible with this version of the specification.
+
+The following exchange protocol values are defined by this specification:
+
+* `openid4vci-v1`
+
+## Credential Offer {#dc_api_credential_offer}
+
+The Issuer sends the Credential Offer object to the Digital Credentials API. All of the Credential Offer parameters MAY be used. The Credential Offer object has been extended to include the following parameters:
```

Comment:
```suggestion
The Issuer sends the Credential Offer object to the Digital Credentials API. All of the Credential Offer parameters MAY be used. As some implementations of the DC API may not allow to network calls, the Credential Offer object is extended to include the following parameters:
```

Reply:
I think it's more "to allow all the installed wallets to figure out if they can handle the request without fetching external data that may result in data being leaked without user consent"? Not sure.

Reply:
I think thats probably right as its highlighting why some DC API implementations have this limitation.